File: packages/cli/tests/commands/template/fixtures/custom-commands/expected/typescript/template.ts
import { Template } from 'e2b'

export const template = Template()
  .fromImage('node:18')
  .setUser('root')
  .setWorkdir('/')
  .setWorkdir('/app')
  .copy('server.js', '.')
  .setUser('user')
  .setStartCmd('sudo node server.js', 'curl -f http://localhost:3000/health || exit 1')

File: packages/cli/tests/commands/template/fixtures/custom-commands/expected/python-sync/template.py
from e2b import Template

template = (
    Template()
    .from_image("node:18")
    .set_user("root")
    .set_workdir("/")
    .set_workdir("/app")
    .copy("server.js", ".")
    .set_user("user")
    .set_workdir("/home/user")
    .set_start_cmd("sudo node server.js", "curl -f http://localhost:3000/health || exit 1")
)